medicines/web/src/pages/substance/index.tsx

```diff
@@ -11,7 +11,7 @@ import {
 import { useLocalStorage } from '../../hooks';
 import { IProduct } from '../../model/substance';
 import Events from '../../services/events';
-import { products } from '../../services/graphql-loader';
+import { products } from '../../services/products-loader';
```

Comment:
@sgrowe I was expecting that the `graphql-loader` would expand a little, so that we can make one big request to the graphql endpoint like `substance-loader` does to Azure, to get a big drill-down

Reply:
Ah ok. I've just tried to keep an identical API to the existing substances loader so that switching between them with the feature flag is easy. 

I guess once the flag is removed that might be a good time to add some tool for combining graphql requests though?